device/bpi.conf

```
@@ -8,3 +8,4 @@ UBLDR_LOADADDR=0x42000000
 export PRODUCT_KERNEL=SMP-BPI
 export PRODUCT_TARGET=arm
 export PRODUCT_ARCH=armv6
+export WANTS=arm-gnueabi-binutils qemu-user-static u-boot-bananapi
```

```suggestion
export PRODUCT_WANTS="arm-gnueabi-binutils qemu-user-static u-boot-bananapi"
```

not sure about qemu-user-static, think about native builds. but can always fix later